data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v5.3.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+    <!--
+     For now, the schema must be identical between here and the 
+     virtual node schema (in ../../vnode-crypto/migration/crypto-creation-v1.0.xml)
+     since the crypto processor uses the same code irrespective of where the table resides.
+     
+     We could just include the same changeset files in both cases, but that might confuse readers
+     and liquibase, so instead we have to keep the content in sync.  
+     -->
```

Comment:
I am not sure we need this comment as it refers to internal things (crypto processor).